### ISSUE TITLE
missing the .h5 file after video analysis

### ISSUE THREAD

**Issue by LaSal2090** [DeepLabCut/DeepLabCut#1637]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Bug description

hey,

I installed DLC using the guidelines in your documentation on a workstation. I Copied an existing neural network to a network drive and used it to analyze a mouse tracking video, that is local on the workstation. The analysis works and creates a .csv, but I'm missing the .h5 file to create a labeled video. 
I do the same process steps, using the GUI, on an other computer and the outcome is fine, so I guess my environment and/or installation is wrong. 

Best
Laurenz

### Operating System

Win10

### DeepLabCut version

DLC version 2.2.0.3, downloaded out of the installation guide from the documentation. 

### DeepLabCut mode

single animal

### Device type

GPU, NVIDIA GeForce RTX 3090

### Steps To Reproduce

Config:

Deeplabcut Environment
(DEEPLABCUT)

# Name                    Version                   Build  Channel
7zip                      19.00                h2d74725_2    conda-forge
absl-py                   1.0.0                    pypi_0    pypi
argon2-cffi               20.1.0           py38h2bbff1b_1
astunparse                1.6.3                    pypi_0    pypi
async_generator           1.10               pyhd3eb1b0_0
attrs                     21.2.0             pyhd3eb1b0_0
backcall                  0.2.0              pyhd3eb1b0_0
bleach                    4.0.0              pyhd3eb1b0_0
ca-certificates           2021.10.8            h5b45459_0    conda-forge
cachetools                4.2.4                    pypi_0    pypi
certifi                   2021.10.8                pypi_0    pypi
cffi                      1.15.0           py38h2bbff1b_0
charset-normalizer        2.0.10                   pypi_0    pypi
colorama                  0.4.4              pyhd3eb1b0_0
cudatoolkit               11.3.1               h59b6b97_2
cudatoolkit-dev           11.4.0               h87d3a71_5    conda-forge
cudnn                     8.2.1                cuda11.3_0
curl                      7.80.0               h2bbff1b_0
cycler                    0.11.0                   pypi_0    pypi
debugpy                   1.5.1            py38hd77b12b_0
decorator                 5.1.0              pyhd3eb1b0_0
deeplabcut                2.2.0.3                  pypi_0    pypi
defusedxml                0.7.1              pyhd3eb1b0_0
entrypoints               0.3                      py38_0
ffmpeg                    4.3.1                ha925a31_0    conda-forge
filterpy                  1.4.5                    pypi_0    pypi
flatbuffers               2.0                      pypi_0    pypi
fonttools                 4.28.5                   pypi_0    pypi
gast                      0.4.0                    pypi_0    pypi
gitdb                     4.0.9                    pypi_0    pypi
gitpython                 3.1.24                   pypi_0    pypi
google-auth               2.3.3                    pypi_0    pypi
google-auth-oauthlib      0.4.6                    pypi_0    pypi
google-pasta              0.2.0                    pypi_0    pypi
grpcio                    1.43.0                   pypi_0    pypi
h5py                      3.6.0                    pypi_0    pypi
icu                       58.2                 ha925a31_3
idna                      3.3                      pypi_0    pypi
imageio                   2.13.5                   pypi_0    pypi
imageio-ffmpeg            0.4.5                    pypi_0    pypi
imgaug                    0.4.0                    pypi_0    pypi
importlib-metadata        4.8.2            py38haa95532_0
importlib_metadata        4.8.2                hd3eb1b0_0
ipykernel                 6.4.1            py38haa95532_1
ipython                   7.29.0           py38hd4e2768_0
ipython_genutils          0.2.0              pyhd3eb1b0_1
ipywidgets                7.6.5              pyhd3eb1b0_1
jedi                      0.18.0           py38haa95532_1
jinja2                    3.0.2              pyhd3eb1b0_0
joblib                    1.1.0                    pypi_0    pypi
jpeg                      9d                   h2bbff1b_0
jsonschema                3.2.0              pyhd3eb1b0_2
jupyter                   1.0.0            py38haa244fe_7    conda-forge
jupyter_client            7.1.0              pyhd3eb1b0_0
jupyter_console           6.4.0              pyhd3eb1b0_0
jupyter_core              4.9.1            py38haa95532_0
jupyterlab_pygments       0.1.2                      py_0
jupyterlab_widgets        1.0.0              pyhd3eb1b0_1
keras                     2.7.0                    pypi_0    pypi
keras-preprocessing       1.1.2                    pypi_0    pypi
kiwisolver                1.3.2                    pypi_0    pypi
libclang                  12.0.0                   pypi_0    pypi
libcurl                   7.80.0               h86230a5_0
libpng                    1.6.37               h2a8f88b_0
libssh2                   1.10.0               h680486a_2    conda-forge
llvmlite                  0.37.0                   pypi_0    pypi
m2w64-gcc-libgfortran     5.3.0                         6
m2w64-gcc-libs            5.3.0                         7
m2w64-gcc-libs-core       5.3.0                         7
m2w64-gmp                 6.1.0                         2
m2w64-libwinpthread-git   5.0.0.4634.697f757               2
markdown                  3.3.6                    pypi_0    pypi
markupsafe                2.0.1            py38h2bbff1b_0
matplotlib                3.5.1                    pypi_0    pypi
matplotlib-inline         0.1.2              pyhd3eb1b0_2
mistune                   0.8.4           py38he774522_1000
moviepy                   1.0.3                    pypi_0    pypi
msgpack                   1.0.3                    pypi_0    pypi
msgpack-numpy             0.4.7.1                  pypi_0    pypi
msys2-conda-epoch         20160418                      1
nb_conda                  2.2.1            py38haa244fe_4    conda-forge
nb_conda_kernels          2.3.1            py38haa95532_0
nbclient                  0.5.3              pyhd3eb1b0_0
nbconvert                 6.1.0            py38haa95532_0
nbformat                  5.1.3              pyhd3eb1b0_0
nest-asyncio              1.5.1              pyhd3eb1b0_0
networkx                  2.6.3                    pypi_0    pypi
notebook                  6.4.6            py38haa95532_0
numba                     0.54.1                   pypi_0    pypi
numexpr                   2.8.1                    pypi_0    pypi
numpy                     1.20.3                   pypi_0    pypi
oauthlib                  3.1.1                    pypi_0    pypi
opencv-python             4.5.5.62                 pypi_0    pypi
opencv-python-headless    4.5.5.62                 pypi_0    pypi
openssl                   1.1.1l               h8ffe710_0    conda-forge
opt-einsum                3.3.0                    pypi_0    pypi
packaging                 21.3               pyhd3eb1b0_0
pandas                    1.3.5                    pypi_0    pypi
pandocfilters             1.4.3            py38haa95532_1
parso                     0.8.3              pyhd3eb1b0_0
patsy                     0.5.2                    pypi_0    pypi
pickleshare               0.7.5           pyhd3eb1b0_1003
pillow                    9.0.0                    pypi_0    pypi
pip                       21.3.1             pyhd8ed1ab_0    conda-forge
proglog                   0.1.9                    pypi_0    pypi
prometheus_client         0.12.0             pyhd3eb1b0_0
prompt-toolkit            3.0.20             pyhd3eb1b0_0
prompt_toolkit            3.0.20               hd3eb1b0_0
protobuf                  3.19.1                   pypi_0    pypi
psutil                    5.9.0                    pypi_0    pypi
pyasn1                    0.4.8                    pypi_0    pypi
pyasn1-modules            0.2.8                    pypi_0    pypi
pycparser                 2.21               pyhd3eb1b0_0
pygments                  2.10.0             pyhd3eb1b0_0
pyparsing                 3.0.4              pyhd3eb1b0_0
pyqt                      5.9.2            py38ha925a31_4
pyrsistent                0.18.0           py38h196d8e1_0
python                    3.8.12          h7840368_2_cpython    conda-forge
python-dateutil           2.8.2              pyhd3eb1b0_0
python_abi                3.8                      2_cp38    conda-forge
pytz                      2021.3                   pypi_0    pypi
pywavelets                1.2.0                    pypi_0    pypi
pywin32                   302              py38h827c3e9_1
pywinpty                  0.5.7                    py38_0
pyyaml                    6.0                      pypi_0    pypi
pyzmq                     22.3.0           py38hd77b12b_2
qt                        5.9.7            vc14h73c81de_0
qtconsole                 5.1.1              pyhd3eb1b0_0
qtpy                      1.10.0             pyhd3eb1b0_0
requests                  2.27.0                   pypi_0    pypi
requests-oauthlib         1.3.0                    pypi_0    pypi
rsa                       4.8                      pypi_0    pypi
ruamel-yaml               0.17.20                  pypi_0    pypi
ruamel-yaml-clib          0.2.6                    pypi_0    pypi
scikit-image              0.18.1                   pypi_0    pypi
scikit-learn              1.0.2                    pypi_0    pypi
scipy                     1.7.3                    pypi_0    pypi
send2trash                1.8.0              pyhd3eb1b0_1
setuptools                58.0.4           py38haa95532_0
shapely                   1.8.0                    pypi_0    pypi
sip                       4.19.13          py38ha925a31_0
six                       1.16.0             pyhd3eb1b0_0
smmap                     5.0.0                    pypi_0    pypi
sqlite                    3.37.0               h2bbff1b_0
statsmodels               0.13.1                   pypi_0    pypi
tables                    3.7.0                    pypi_0    pypi
tabulate                  0.8.9                    pypi_0    pypi
tensorboard               2.7.0                    pypi_0    pypi
tensorboard-data-server   0.6.1                    pypi_0    pypi
tensorboard-plugin-wit    1.8.0                    pypi_0    pypi
tensorflow                2.7.0                    pypi_0    pypi
tensorflow-estimator      2.7.0                    pypi_0    pypi
tensorflow-io-gcs-filesystem 0.23.1                   pypi_0    pypi
tensorpack                0.11                     pypi_0    pypi
termcolor                 1.1.0                    pypi_0    pypi
terminado                 0.9.4            py38haa95532_0
testpath                  0.5.0              pyhd3eb1b0_0
tf-slim                   1.1.0                    pypi_0    pypi
threadpoolctl             3.0.0                    pypi_0    pypi
tifffile                  2021.11.2                pypi_0    pypi
tornado                   6.1              py38h2bbff1b_0
tqdm                      4.62.3                   pypi_0    pypi
traitlets                 5.1.1              pyhd3eb1b0_0
typing-extensions         4.0.1                    pypi_0    pypi
urllib3                   1.26.7                   pypi_0    pypi
vc                        14.2                 h21ff451_1
vs2015_runtime            14.27.29016          h5e58377_2
wcwidth                   0.2.5              pyhd3eb1b0_0
webencodings              0.5.1                    py38_1
werkzeug                  2.0.2                    pypi_0    pypi
wheel                     0.37.0             pyhd3eb1b0_1
widgetsnbextension        3.5.1                    py38_0
wincertstore              0.2              py38haa95532_2
winpty                    0.4.3                         4
wrapt                     1.13.3                   pypi_0    pypi
wxpython                  4.0.7.post2              pypi_0    pypi
zipp                      3.6.0              pyhd3eb1b0_0
zlib                      1.2.11               h8cc25b3_4

### Relevant log output

```shell
(DEEPLABCUT) PS C:\> python -m DEEPLABCUT
C:\Users\salm0001\Anaconda3\envs\DEEPLABCUT\python.exe: No module named DEEPLABCUT
(DEEPLABCUT) PS C:\> python -m deeplabcut
Starting GUI...
Using snapshot-200000 for model U:\RG Donato\Group\Laurenz\DLC\open_arena_blackNBZ_floor_test-Steffen-2020-09-08\dlc-models\iteration-1\open_arena_white_floorSep8-trainset95shuffle1
C:\Users\salm0001\Anaconda3\envs\DEEPLABCUT\lib\site-packages\tensorflow\python\keras\engine\base_layer_v1.py:1694: UserWarning: `layer.apply` is deprecated and will be removed in a future version. Please use `layer.__call__` method instead.
  warnings.warn('`layer.apply` is deprecated and '
2022-01-11 12:19:25.094562: I tensorflow/core/platform/cpu_feature_guard.cc:151] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  AVX AVX2
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
2022-01-11 12:19:25.607830: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1525] Created device /job:localhost/replica:0/task:0/device:GPU:0 with 21672 MB memory:  -> device: 0, name: NVIDIA GeForce RTX 3090, pci bus id: 0000:17:00.0, compute capability: 8.6
Starting to analyze %  D:\temp\DON-007050_20211031_TR-BSL_NS1-ACQ(crop).mp4
Loading  D:\temp\DON-007050_20211031_TR-BSL_NS1-ACQ(crop).mp4
Duration of video [s]:  300.0 , recorded with  20.0 fps!
Overall # of frames:  6000  found with (before cropping) frame dimensions:  2064 1544
Starting to extract posture
  0%|                                                                                         | 0/6000 [00:00<?, ?it/s]2022-01-11 12:19:29.819511: I tensorflow/stream_executor/cuda/cuda_dnn.cc:366] Loaded cuDNN version 8201
6060it [07:22, 13.69it/s]
Saving results in D:\temp...
Saving csv poses!
The videos are analyzed. Now your research can truly start!
 You can create labeled videos with 'create_labeled_video'
If the tracking is not satisfactory for some videos, consider expanding the training set. You can use the function 'extract_outlier_frames' to extract a few representative outlier frames.
Starting to process video: D:\temp\DON-007050_20211031_TR-BSL_NS1-ACQ(crop).mp4
Loading D:\temp\DON-007050_20211031_TR-BSL_NS1-ACQ(crop).mp4 and data.
No unfiltered data file found in D:\temp for video DON-007050_20211031_TR-BSL_NS1-ACQ(crop) and scorer DLC_resnet50_open_arena_white_floorSep8shuffle1_200000.
```

### Anything else?

A similar case was reported here: https://github.com/DeepLabCut/DeepLabCut/issues/1635
but the solution did not fit for me. 

I also got the Error: Failed to launch ptxas, as described in the Issue above, but the solution helped to solve that. 

Any similar feeds that stated the same issues, did not gave any working solution for me. 
Any help is much appreciated.

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/DeepLabCut/DeepLabCut/blob/master/CODE_OF_CONDUCT.md)

**Comment by MMathisLab** [comment]
could it be a writing permission error? You are launching DLC from C drive, but attempting to write h5 into D; double check permissions (if not done already), also try to upgrade `pytables`

**Comment by LaSal2090** [comment]
Hi,
thanks for your reply. I checked the permissions and changed them accordingly, that my user account has all rights to write. Further I tried to analyze the video on the C: drive, so that DLC and writing folder are at the same drive.
Both changes did not help with my problem. Thanks for pointing out the pytables update

**Comment by jeylau** [comment]
@LaSal2090, you may need to install specifically this version `pip install tables==3.6.1`

**Comment by LaSal2090** [comment]
That did the trick! Because there was no error massage showing, I was totally lost.
Thank you so much.
